### DISCUSSION TITLE
Draft poster for Wikimania 2024

### DISCUSSION THREAD

**Discussion by Ainali** [govdirectory/brand-resources#9] [Ideas]
Here is a first complete draft for the Wikimania poster [Add your country to the Wikidata Govdirectory](https://wikimania.eventyay.com/2024/talk/review/ZSKWHBUEFLA3EW9ZDPDCCCVDWKKR9HSM).

Have I missed something important? Is there something out of date that should be addressed? Do we want something more in the footer? Ping @Abbe98 and @RVA2869 

![govdirectory-posterdraft](https://github.com/govdirectory/brand-resources/assets/5566159/8d5c2103-e1de-4469-b843-35fdfbfa75f7)

**Comment by RVA2869** [comment]
What is the purpose of this poster? But that's a lot of text.

**Reply by Ainali** [reply]
The purpose is to provide an insight in how to add a new country.

**Reply by RVA2869** [reply]
> The purpose is to provide an insight in how to add a new country.

So we assume that the participants are already familiar with Govdirectory?

**Reply by Ainali** [reply]
> So we assume that the participants are already familiar with Govdirectory?

Not at all. What more, if anything, do you think is needed to explain what it is?

**Reply by RVA2869** [reply]
> > So we assume that the participants are already familiar with Govdirectory?
> 
> Not at all. What more, if anything, do you think is needed to explain what it is?

Always assume that people don't know what you're talking about. (Especially at such large events)

**Reply by Ainali** [reply]
> Always assume that people don't know what you're talking about. (Especially at such large events)

That is precisely what I said. So I'll repeat: What more, if anything, do you think is needed to explain what it is?

**Reply by RVA2869** [reply]
> That is precisely what I said.

I know 😉 . 

Maybe 50/50, the first 50% of the poster a little explanation about Govdirectory and the other 50 about the how to but keep it "basic" (if there are people who are interested in the project they will find the extensive explanation on the project page).

**Reply by Ainali** [reply]
I don't understand what you mean with keeping it "basic". Could you elaborate?
(Also, we are not trying to duplicate the poster [that was on Wikimania last year](https://commons.wikimedia.org/wiki/File:Govdirectory-Wikimania2023-poster.pdf).)

**Reply by RVA2869** [reply]
> I don't understand what you mean with keeping it "basic". Could you elaborate

Maybe it's just me, but a poster with a rather large block of text is not inviting to potential new members.
Basic in the sense of: keep it light – don't dive too deep. Short lists with some graphics. But I also don't know how long this "keynote" should/can last?

**Reply by Ainali** [reply]
This *is* fairly light for a conference poster. :smile: 
Do a quick image search and you will see plenty of more information dense examples.

**Comment by RVA2869** [comment]
A small change in the footer: `Mastodon:@govdirectory@wikis.world `

**Reply by Ainali** [reply]
Or should it be `Fediverse:`?

**Comment by RVA2869** [comment]
Maybe incorporate a QR-code with the video "[Add country to WikiProject Govdirectory](https://commons.wikimedia.org/wiki/File:Add_country_to_WikiProject_Govdirectory.webm)" (just an idea)

**Reply by Ainali** [reply]
Good idea! And maybe one to the project page too?

**Comment by Ainali** [comment]
This was the final version: https://commons.wikimedia.org/wiki/File:Govdirectory-add-your-country-poster.pdf

It led to a couple of good discussions, and I got some new sources for the Philippines and Denmark. I'll add them to the country pages. Unfortunately, it was taped to the wall, so I failed to bring it home in a good-looking piece to be able to reuse it.
